### DISCUSSION TITLE
v2.5.0

### DISCUSSION THREAD

**Discussion by Zenda-Cross** [vega-org/vega-app#139] [Announcements]
## Note- Uninstall the Previous Version of the app before Installing v2.5.0
- Improved and fixed MultiStream and NetflixMirror
- Removed dead providers.
- Security fix

<hr /><em>This discussion was created from the release <a href='https://github.com/Zenda-Cross/vega-app/releases/tag/v2.5.0'>v2.5.0</a>.</em>

**Comment by adilurrhmn** [comment]
1080p is not available in Netflix mirror

**Reply by Zenda-Cross** [reply]
Okay I will try to fix that
